nms/app/packages/magmalte/app/views/equipment/EnodebDetailMain.js

```diff
@@ -101,51 +68,58 @@ export function EnodebDetail() {
 
   return (
     <>
-      <div className={classes.topBar}>
-        <Text variant="body2">Equipment/{enbInfo.enb.name}</Text>
-      </div>
-
-      <AppBar position="static" color="default" className={classes.tabBar}>
-        <Grid container direction="row" justify="flex-end" alignItems="center">
-          <Grid item xs={6}>
-            <Tabs
-              value={GetCurrentTabPos(match.url, ['overview', 'config'])}
-              indicatorColor="primary"
-              TabIndicatorProps={{style: {height: '5px'}}}
-              textColor="inherit"
-              className={classes.tabs}>
-              <Tab
-                key="Overview"
-                component={NestedRouteLink}
-                label={<OverviewTabLabel />}
-                to="/overview"
-                className={classes.tab}
-              />
-              <Tab
-                key="Config"
-                component={NestedRouteLink}
-                label={<ConfigTabLabel />}
-                to="/config"
-                className={classes.tab}
-              />
-            </Tabs>
-          </Grid>
-          <Grid item xs={6}>
-            <Grid container justify="flex-end" alignItems="center" spacing={2}>
-              <Grid item>
-                <Button className={classes.appBarBtnSecondary}>
-                  Secondary Action
-                </Button>
+      <TopBar
+        header={`Equipment/${enbInfo.enb.name}`}
+        tabs={[
+          {
+            label: 'Overview',
+            to: '/overview',
+            icon: DashboardIcon,
+            filters: (
+              <Grid
+                container
+                justify="flex-end"
+                alignItems="center"
+                spacing={2}>
+                <Grid item>
+                  <Button className={classes.appBarBtnSecondary}>
+                    Secondary Action
+                  </Button>
+                </Grid>
```

Comment:
@ekfuhrmann This looks like older code. I recently merged some changes which added reboot button and got rid of secondary action here